tests/rptest/tests/controller_erase_test.py

```diff
@@ -86,11 +89,19 @@ def wait_all_segments():
                 "0_0"].segments.keys()
             assert len(segments) == transfers_leadership_count + 1
             segments = sorted(list(segments))
-            victim_path = f"{controller_path}/{segments[-1]}.log"
+
```

Comment:
The intent of the test is to have all the data replicated: in the case where one node is very slow (like in the failure we saw), we should wait for the slow node to catch up (and apply all controller messages) rather than modifying the deletion procedure.